kairon/chat/handlers/channels/clients/whatsapp/cloud.py

```
@@ -185,6 +185,10 @@ def mark_as_read(self, msg_id, timeout=None):
         payload = {"messaging_product": "whatsapp", "status": "read", "message_id": msg_id}
         return self.send_action(payload)
 
+    def typing_indicator(self, msg_id, timeout=None):
```

Ruff (0.14.5)

Unused method argument: `timeout`

(ARG002)